backend-peer/src/lib/error-handler/index.ts

```diff
@@ -32,11 +32,20 @@ interface IInternalError {
 export class HttpResponse extends Error {
   public code: number
   public message: string
-
-  constructor({ code = 500, message = 'Internal server error' }) {
+  public details: string
+  public trace: string
+
+  constructor({
+    code = 500,
+    message = 'Internal server error',
+    details = '',
+    trace = '',
+  }) {
     super(message)
     this.code = code
     this.message = message
+    this.details = details
+    this.trace = trace
```

Comment:
I couldn't find a case where a trace was present in a response from cloudagent? So maybe we don't need this